plugins/module_utils/fusion.py

```diff
@@ -112,10 +111,6 @@ def get_fusion(module):
         module.fail_on_missing_params([PARAM_PRIVATE_KEY_FILE])
 
     config = fusion.Configuration()
-    if ENV_API_HOST in environ or ENV_HOST in environ:
-        host_url = environ.get(ENV_API_HOST, environ.get(ENV_HOST))
-        config.host = urljoin(host_url, BASE_PATH)
-    config.token_endpoint = environ.get(ENV_TOKEN_ENDPOINT, config.token_endpoint)
 
     if access_token is not None:
```

Comment:
I think you can even remove these if statements for `access_token`, `private_key_password`, `private_key_file` and just put all of them into fusion.Configuration() initialization

Reply:
Done in [88d10ac](https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/140/commits/88d10acf05094bfb848ab8ade89095d6ec09d52c)